src/cpp/src/llm_pipeline.cpp

```diff
@@ -14,6 +14,24 @@
 #include "utils.hpp"
 #include "text_callback_streamer.hpp"
 
+namespace {
+
+ov::genai::TokenizedInputs subtract_chat_tokenized_inputs(const ov::genai::TokenizedInputs& fisrt, const ov::genai::TokenizedInputs& second){
+    auto first_size = fisrt.input_ids.get_size();
+    auto second_size = second.input_ids.get_size();
+    ov::Shape new_shape{1, first_size - second_size};
+
+    ov::Tensor new_input_ids(ov::element::i64, new_shape);
+    auto data_ptr = fisrt.input_ids.data<int64_t>();
+    std::copy(data_ptr + second_size, data_ptr + first_size, new_input_ids.data<int64_t>());
+
+    ov::Tensor new_attention_mask(ov::element::i64, new_shape);
+    std::fill_n(new_attention_mask.data<int64_t>(), new_shape[1], 1);
```

Comment:
can you return view tensors to `fisrt` instead of copy

Reply:
original tensor with real memory will be dead before this view tensors is used

Reply:
My expectation was that such tensor prolongs the lifetime of the original tensor. This seems to be the truth because the underluing RoiTensor stores the original tensor in `make_tensor()`.
```C++
Tensor::Tensor(const Tensor& owner, const Coordinate& begin, const Coordinate& end)
    : _impl{make_tensor(owner._impl, begin, end)},
      _so{owner._so} {}
```
Isn't that correct?

Reply:
I thought you're talking about another view tensor Tensor(const ov::Output<const ov::Node>& port, void* host_ptr...)
Let me try this one with Coordinate

Reply:
CVS-146098 - ROI tensor issue